src/App.js

```diff
@@ -1,10 +1,11 @@
+import CurrencyExchange from './components/CurrencyExchange';
+import Navbar from './components/Navbar';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <h1>HELLo</h1>
-      </header>
+      <Navbar />
```

Comment:
- [ ] Good job working on this app, but you did not quite follow the project and design requirements. Kindly reference the below screenshots for the design and project requirements. Please kindly ensure the project flow is implemented as per the instructions

### Project Core Requirements
![image](https://github.com/AbbasSarwar/CurrencyExchange/assets/43995830/46ae2419-b67b-41fe-a757-ed4ce17a2b6b)


### Design
![image](https://github.com/microverseinc/curriculum-react-redux/raw/main/capstone/images/metrics_app.png)